euth/blueprints/templates/euth_blueprints/includes/infopage.html

```diff
@@ -12,39 +12,25 @@ <h2 class="sans-serif h4">{{ blueprint.title }}</h2>
 
         <div class="blueprint-description border-bottom">
             {{ blueprint.description | linebreaks }}
-            <a href="{% url 'dashboard-project-create' organisation_slug=view.organisation.slug blueprint_slug=blueprint_slug %}" class="btn btn-primary">
+            <a href="{% url 'dashboard-project-create' organisation_slug=view.organisation.slug blueprint_slug=blueprint_slug %}" class="btn btn-primary" target="_blank">
             {% trans 'Use this template' %}
             </a>
         </div>
 
         <div class="blueprint-phases border-bottom">
             <h4>{% trans 'Project Phases' %}</h4>
 
-            <ul class="list-inline">
+            <ul class="list-unstyled">
                 {% for phase_content in blueprint.content %}
-                    <li class="blueprint-phases-item">
-                        <span class="fa-stack fa-4x" aria-hidden="true">
-                            <i class="fa fa-circle fa-stack-2x text-brand5"></i>
-                            <i class="fa fa-{{ phase_content.icon }} fa-stack-1x text-brand6"></i>
-                        </span>
-                        <p>
-                            <strong>
-                                {{ forloop.counter }}. {{ phase_content.description }}</p>
-                            </strong>
-                        </p>
+                    <li>
+                          <div class="icon">
+                            <span class="circled"><i class="fa fa-{{ phase_content.icon }} fa-fw" aria-hidden="true"></i>
+                          </div>
+                          <div class="description">
+                            <strong>Phase {{ forloop.counter }}</strong><p>{{ phase_content.description }}</p>
+                          </div>
```

Comment:
Oh, added in f81543f7a5791abdcf2c282dc4d3298f1498654a. The hard coded way seems ugly